doc/source/contributing.md

```diff
@@ -0,0 +1,137 @@
+
+## Contribute to documentation yourself
+
+To contribute to documentation, learn about the following:
+- [reStructured text](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) markup language. It is used to write the documentation.
+- [Sphinx](https://www.sphinx-doc.org/en/master/usage/quickstart.html) documentation generator. We use it to convert source ``.rst`` files to html and PDF documents.
+- [git](https://git-scm.com/)
+- [Docker](https://docs.docker.com/get-docker/). It allows you to run Sphinx in a virtual environment instead of installing it and its dependencies on your machine.
+
+There are several active versions of the documentation. Each version has a branch in the repository named accordingly:
+- 5.7
+- 8.0
+
+The .rst files are in the ``source`` directory. 
+
+### Edit documentation online via GitHub
+
+1. Click the **Edit this page** link on the sidebar. The source ``.rst`` file of the page opens in GitHub editor in your browser. If you haven’t worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of it for you.
+
+2. Edit the page. You can check your changes on the **Preview** tab.
+
+   **NOTE**: GitHub’s native markup language is [Markdown](https://daringfireball.net/projects/markdown/) which differs from the reStructured text. Therefore, though GitHub renders titles, headings and lists properly, some rst-specific elements like variables, directives, internal links are not rendered.
+
+3. Commit your changes.
+
+	 - In the *Commit changes* section, describe your changes.
+	 - Select the **Create a new branch for this commit and start a pull request** option
+	 - Click **Propose changes**.
+
+4. GitHub creates a branch and a commit for your changes. It loads a new page on which you can open a pull request to Percona. The page shows the base branch - the one you offer your changes for, your commit message and a diff - a visual representation of your changes against the original page. At this time you can make a last-minute review. When you are ready, click the **Create pull request** button.
+5. Someone from our team reviews the pull request for accuracy and if everything is correct, merges it into the documentation. Then it gets published on the site in the future.
+
+### Edit documentation locally
+
+This option is for users who prefer to work from their computer and/or have the full control over the documentation process.
+
+The steps are the following:
+
+1. Fork this repository
+2. Clone the repository on your machine:
+
+```sh
+git clone git@github.com:<your_name>/percona-server.git
+```
+
+3. Change the directory to ``percona-server/docs`` and add the remote upstream repository:
+
+```sh
+git remote add upstream git@github.com:percona/percona-server.git
```

Comment:
I believe percona/percona-server = origin
here and below this should be fixed

Reply:
Fixed.